From 268c939766aba7da3764c9529cc7e7e793c5fcfd Mon Sep 17 00:00:00 2001
From: VladislavBryl <bril.vladislav@gmail.com>
Date: Fri, 17 Sep 2021 13:17:50 +0300
Subject: [PATCH] fix truncation error

---
 calculator.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/calculator.cpp b/calculator.cpp
index b91afea..498f7b9 100644
--- a/calculator.cpp
+++ b/calculator.cpp
@@ -2,7 +2,7 @@
 
 int Calculator::Add (double a, double b)
 {
-    return a + b;
+    return a + b + 0.5;
 }
 
 int Calculator::Sub (double a, double b)
-- 
2.31.1.windows.1

